### PR TITLE
fix: dissolve other + fixes

### LINKED ISSUE #1127: Dissolve "Other" category
Move the documents in the "Other category" somewhere where users can find them. 

My proposal: 

- Make 
Ory CLI 
a top level category 

Alternative: Top Menu Bar, next to API, but it is getting crowded there

- Custom UI w/ Ory Elements (or full: Learn how to create custom TypeScript SPA with Ory Elements)
~Rename to "TypeScript SPA with Ory Elements" in the sidebar and~ put under Quickstart/Login. 

Alternative: Rename to "TypeScript SPA" and add under a "Ory Elements/Elements" Quickstarts category

- Terminology
This document can be dissolved in "Introduction" and other introductory documents i.e. the terms should be explained in the first document you read where the term is used

Alternative: Move to Troubleshooting? Or after Introduction? It could also fit in a reference category. 

- Browser redirects and flow completion

Move to "self-service/flows/", as it only concerns the identities part and self-service flows

Alternative: Move somewhere else prominently in the Identities and sessions section.

### PR DESCRIPTION
**Pull request by vinckr**
This fixes the issues mentioned in https://github.com/ory/docs/issues/1127 +
- rewrite introduction and delete/move outdated & duplicate content
- remove concepts/project doc, content got moved to intro or deleted bc duplicate
- create Keto selfhosted section and move docs only relevant for self-hosted there
- switch places Hydra & Kratos selfhosted to mirror structure in main doc

closes #1127

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [x] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: src/sidebar.js [hunk 1]
```diff
@@ -175,7 +183,7 @@ module.exports = {
     },
     {
       type: "category",
-      label: "OAuth2 & OpenID",
+      label: "OAuth2 and OIDC",
```
Comment (aeneasr) [inline src/sidebar.js:186]: The product is called: Ory OAuth2 and OpenID, see: https://docs.google.com/document/d/1GWNnI4dyVlXZzKMLPMPiuvz3rQJHf_P_jg3flsUq-ec/edit

File: src/sidebar.js [hunk 2]
```diff
@@ -22,6 +22,7 @@ module.exports = {
             "getting-started/integrate-auth/react",
             "getting-started/integrate-auth/vue",
             "getting-started/integrate-auth/react-native",
+            "getting-started/custom-ui-ory-elements",
```
Comment (aeneasr) [inline src/sidebar.js]: Move this to bring your own ui please

File: docs/welcome.mdx [hunk 3]
```diff
@@ -5,31 +5,31 @@ sidebar_label: Introduction
 ---
```
Comment (aeneasr) [inline docs/welcome.mdx:5]: Let's make these changes in a separate PR with input from Klaus who already has a plan for the contents of this page. You can find a draft here: https://www.notion.so/Introduction-38cc979e1cd444cb99a5a19318283c88

File: vercel.json [hunk 4]
```diff
@@ -136,7 +136,7 @@
     },
     {
       "source": "/docs/start-building/create-project",
-      "destination": "https://www.ory.sh/docs/guides/console/create-project",
```
Comment (aeneasr) [inline vercel.json:139]: Please instead create a new redirect for `/docs/guides/console/create-projec`

File: vercel.json [hunk 5]
```diff
@@ -146,7 +146,7 @@
     },
     {
       "source": "/docs/start-building/ory-cli-install-use",
-      "destination": "https://www.ory.sh/docs/guides/ory-cli-install-use",
+      "destination": "https://www.ory.sh/docs/guides/cli/installation",
```
Comment (aeneasr) [inline vercel.json:149]: Same here, please create a new redirect instead. Otherwise https://www.ory.sh/docs/guides/ory-cli-install-use will not work.

**Review by aeneasr — CHANGES_REQUESTED** [review]

File: docs/welcome.mdx (hunk 3 above)
Comment (aeneasr) [reply docs/welcome.mdx:5]: the changes are fine tho

File: vercel.json (hunk 4 above)
Comment (vinckr) [reply vercel.json]: I reverted this change, is probably better to address in a dedicated PR!

File: src/sidebar.js (hunk 1 above)
Comment (aeneasr) [reply src/sidebar.js:186]: OpenID is a recognized name, experts know what OIDC means but many do not. Let's please stick to the product name here. We can use the 100% accurate terminology in the documentation:

```suggestion
      label: "OAuth2 and OpenID",
```

File: vercel.json [hunk 6]
```diff
@@ -445,10 +445,20 @@
...
+      "permanent": false
+    },
+    {
+      "source": "/docs/concepts/terminology",
       "destination": "https://www.ory.sh/docs/welcome",
       "permanent": false
     },
+    {
+      "source": "/docs/guides/console/create-project",
+      "destination": "https://www.ory.sh/docs/guides/console/create-project",
```
Comment (aeneasr) [inline vercel.json:459]: Bruh you're now redirecting `https://www.ory.sh/docs/guides/console/create-project` to `https://www.ory.sh/docs/guides/console/create-project` ...

**Review by aeneasr — CHANGES_REQUESTED** [review]
please fix the redirects

**Comment by vinckr** [comment]
fixed now 🙏

File: docs/welcome.mdx [hunk 7]
```diff
@@ -5,31 +5,31 @@ sidebar_label: Introduction
...
 
 Ory is a software infrastructure provider building a global zero-trust network for humans, robots, devices, and software services.
-Ory develops open-source software on [GitHub](https://github.com/ory) and publishes open standards such as the
+Ory develops open source software on [GitHub](https://github.com/ory) and publishes open standards such as the
```
Comment (tomekpapiernik) [inline docs/welcome.mdx:8]: `open-source` here is an adjective - why did you remove the hyphen?

File: docs/welcome.mdx [hunk 8]
```diff
@@ -5,31 +5,31 @@ sidebar_label: Introduction
...
 [Ory Permission Language](https://github.com/ory/keto/blob/master/docs/ory_permission_language_spec.md).
-[The Ory Network](https://console.ory.sh/) uses cloud-native open-source technologies (Kubernetes, Crossplane, Cockroach, Linux,
+[The Ory Network](https://console.ory.sh/) uses cloud-native open source technologies (Kubernetes, Crossplane, Cockroach, Linux,
```
Comment (tomekpapiernik) [inline docs/welcome.mdx:10]: `open-source` here is an adjective - why did you remove the hyphen?

File: docs/welcome.mdx [hunk 9]
```diff
@@ -5,31 +5,31 @@ sidebar_label: Introduction
...
+Core infrastructure components of [Ory Network](https://console.ory.sh) are open source to foster collaboration, reduce supply
 chain risk, broaden access to secure services, and introduce the open standard for internet security. Being open source Ory
 improves the safety of everyone:
 
-- Ory Identities offers a secure and modern central identity management solution with MFA, paswordless, WebAuthn, and more. It is
-  based on the open-source [Ory Kratos Identity Server](https://github.com/ory/kratos).
-- Ory OAuth2 & OpenID Connect implements 15+ IETF and OpenID standards to facilitate single-sign on, delegation and API access
-  authorization. It is based on the open-source [Ory Hydra Federation Server](https://github.com/ory/hydra). implementing 15+ IETF
+- Ory Identities offers a secure and modern central identity management solution with MFA, passwordless, WebAuthn, and more. It is
+  based on the open source [Ory Kratos Identity Server](https://github.com/ory/kratos).
```
Comment (tomekpapiernik) [inline docs/welcome.mdx:20]: `open-source` here is an adjective - why did you remove the hyphen?

File: docs/welcome.mdx [hunk 10]
```diff
@@ -5,31 +5,31 @@ sidebar_label: Introduction
...
+- Ory OAuth2 & OpenID Connect implements 15+ IETF and OpenID standards to facilitate single-sign-on, delegation, and API access
```
Comment (tomekpapiernik) [inline docs/welcome.mdx:21]: it's `single sign-on`
https://en.wikipedia.org/wiki/Single_sign-on

File: docs/welcome.mdx [hunk 11]
```diff
@@ -5,31 +5,31 @@ sidebar_label: Introduction
...
+  authorization. It is based on the open source [Ory Hydra Federation Server](https://github.com/ory/hydra). implementing 15+ IETF
```
Comment (tomekpapiernik) [inline docs/welcome.mdx:22]: `open-source` here is an adjective - why did you remove the hyphen?

File: docs/welcome.mdx [hunk 12]
```diff
@@ -5,31 +5,31 @@ sidebar_label: Introduction
...
   and OpenID standards.
 - Ory Permissions is a low-latency, high-performance, relationship-based authorization system that enables fine-grained access
-  control (incl. RBAC and ABAC models) in any application. It is based on the open-source
+  control (incl. RBAC and ABAC models) in any application. It is based on the open source
```
Comment (tomekpapiernik) [inline docs/welcome.mdx:25]: `open-source` here is an adjective - why did you remove the hyphen?

File: docs/welcome.mdx [hunk 13]
```diff
@@ -5,31 +5,31 @@ sidebar_label: Introduction
...
   [Ory Keto Permission Server](https://github.com/ory/keto), which implements
   [Zanzibar: Google’s Consistent, Global Authorization System](https://research.google/pubs/pub48190/).
 
-Ory develops and maintains many additional open-source projects. From an Ory Zero Trust Identity & Access Proxy
+Ory develops and maintains many additional open source projects. From an Ory Zero Trust Identity & Access Proxy
```
Comment (tomekpapiernik) [inline docs/welcome.mdx:29]: `open-source` here is an adjective - why did you remove the hyphen?

File: docs/welcome.mdx [hunk 14]
```diff
@@ -38,140 +38,94 @@ improved hourly.
...
 - From designing Identity Schemas using JSON Schema, to webhooks, to advanced configuration options - Ory is the most customizable
   platform out there.
 - Ory spans the whole authentication and authorization universe with well-designed products and APIs:
-- Identity Management with session management, flows for login, registration, account recovery & verification, mfa, and many more.
+- Identity Management with session management, flows for login, registration, account recovery & verification, MFA, and many more.
 - Permission and Role Management.
 - Delegation via OAuth2 and OpenID Connect.
 - Zero Trust Networking.
-- Modern API design with partial support for gRPC.
+- Modern API Design with partial support for gRPC.
```
Comment (tomekpapiernik) [inline docs/welcome.mdx:51]: why capitalize "design"?

File: docs/welcome.mdx [hunk 15]
```diff
@@ -38,140 +38,94 @@ improved hourly.
...
-  accept tos, shipping address, gender, ...) and create your own login, registration, profile settings, recovery, and verification
-  screen using our easy to use SDKs and REST APIs.
+  accept tos, shipping address, gender, ...) and create your login, registration, profile settings, recovery, and verification
+  screen using SDKs and REST APIs.
 
-### Permission services
+### Permissions and relationships
 
-The Ory Network incorporates the open-source [Ory Keto Permission Server](https://www.ory.sh/keto) project and offers:
+Ory Network incorporates the open source [Ory Keto Permission Server](https://www.ory.sh/keto) and offers:
```
Comment (tomekpapiernik) [inline docs/welcome.mdx:85]: `open-source` here is an adjective - why did you remove the hyphen?

**Review by tomekpapiernik — COMMENTED** [review]
Please pay attention to how you use "open source". 
It must have a hyphen when it's used as an adjective. No hyphen when used as a noun.

File: docs/welcome.mdx (hunk 7 above)
Comment (vinckr) [reply docs/welcome.mdx:8]: My bad, it didnt look right to me 😓 
I will add it again for the adjectives!
We should document it or link to the microsoft styleguide somewhere:
```
Hyphenate open-source as an adjective preceding a noun, as in open-source software. Don't use open-sourced as an adjective.

Don't hyphenate in other instances, such as : "Open source is a development model in which ...."

Don't use OSS as an abbreviation for open-source software.
```

File: docs/welcome.mdx [hunk 16]
```diff
@@ -38,140 +38,94 @@ improved hourly.
...
-The Ory Network incorporates the open-source [Ory Kratos Identity Server](https://www.ory.sh/kratos) project and offers:
+Ory Network incorporates the open-source [Ory Kratos Identity Server](https://www.ory.sh/kratos) and offers:
 
 - **Self-service** flows are everything users do on their own / without the help of others:
-- Registration with passwords, social sign in, OpenID Connect, ...
-- Login with passwords, social sign in, OpenID Connect, ...
-- Updating the profile, email, changing the password, un/linking with Social. Sign In providers, ...
+- Registration with passwords, social sign-in, OpenID Connect, ...
+- Login with passwords, social sign-in, OpenID Connect, ...
+- Updating the profile, and email, changing the password, un/linking with Social. Sign In providers, ...
```
Comment (tomekpapiernik) [inline docs/welcome.mdx:74]:
```suggestion
- Updating the profile, email, changing the password, un/linking with social sign-in providers, ...
```

Comment (tomekpapiernik) [reply docs/welcome.mdx:74]: I can see that Grammarly made you add an "and" here but trust me - it doesn't make sense.

File: docs/welcome.mdx [hunk 17]
```diff
@@ -38,140 +38,94 @@ improved hourly.
...
-  applications and APIs.
-- [Ory Oathkeeper Quickstart.](https://www.ory.sh/docs/oathkeeper/configure-deploy) Authenticate, authorize and mutate any
-  incoming network traffic, with a Zero Trust security model.
-
-Don't want to deal with Docker, Kubernetes, and securing your API?
+### Ory Console
 
-- [Sign up](https://console.ory.sh/registration) for a free Developer account in The Ory Network.
+Ory Console is the management UI of Ory Network. It offers capabilities such as showing and searching for identities, managing
+user sessions, and gaining valuable insights in the form of dashboards and customization.
```
Comment (tomekpapiernik) [inline docs/welcome.mdx:99]: Customization isn't a "valuable insight", is it?
As for dashboards - these aren't in the Console yet.

File: docs/welcome.mdx [hunk 18]
```diff
@@ -38,140 +38,94 @@ improved hourly.
...
+ExpressJS, React, or Preact.
 
-**Read our in-depth concepts documentation**
+## Ory Open Source
 
-- [Identity and User Management (Ory Kratos)](https://www.ory.sh/docs/kratos/concepts/index)
-- [OAuth 2.0 & OpenIDConnect (Ory Hydra)](https://www.ory.sh/hydra/docs/concepts/oauth2)
-- [Authorization and Permission Server (Ory Keto)](https://www.ory.sh/docs/keto/concepts/relation-tuples/)
-- [Zero Trust Networking (Ory Oathkeeper)](https://www.ory.sh/oathkeeper/docs/)
+Ory is the largest open source ecosystem in the area of authentication, authorization, access control, and zero trust networking
```
Comment (tomekpapiernik) [inline docs/welcome.mdx:112]:
```suggestion
Ory is the largest open-source ecosystem in the area of authentication, authorization, access control, and zero-trust networking
```

**Review by tomekpapiernik — CHANGES_REQUESTED** [review]

File: docs/welcome.mdx (hunk 17 above)
Comment (vinckr) [reply docs/welcome.mdx:99]: Agreed. 

I would say in that sentence we dont need more than 

"Ory Console is the management UI of Ory Network."

There is too many features being added all the time that this description would not be outdated very quickly. 
IMO the above keeps it simple and accurate. 
You manage Ory Network through the console, thats about it, no need to write a marketing letter here.

**Pull request merged**